src/modules/requirements/TemporaryBattleRequirement.ts

```diff
@@ -11,6 +11,6 @@ export default class TemporaryBattleRequirement extends Requirement {
     }
 
     public hint(): string {
-        return `Requires beating ${this.battleName}.`;
+        return `Requires beating ${this.battleName.replace(/\s(?=[\d])\d/g, '')}.`;
```

Comment:
Temp battles have a display name. I know this is in modules and temp battles are not, so it's not as easy. But this seems like a temp solution

Reply:
It's definitely a temp solution haha, I have no idea how to reference the displayName from scripts here...

Reply:
We will leave this like this for now...